keps/sig-windows/20190418-windows-security-context.md

```diff
@@ -112,7 +175,16 @@ type WindowsSecurityOptions struct {
 }
 ```
 
-Field `WindowsOptions *WindowsSecurityOptions` will be added to `SecurityContext` and `PodSecurityContext` structs.
+Field `WindowsOptions *WindowsSecurityContextOptions` will be added to `SecurityContext` and `PodSecurityContext` structs.
+
+#### Validation of fields
+
+The fields within the `WindowsSecurityContextOptions` struct will be validated as follows:
+- GMSACredentialSpecName: This field will be the name of a custom resource. It should follow the rules associated with [naming of resources](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names) in Kubernetes. Only the length will be checked to make sure it is below 64 characters.
+- GMSACredentialSpec: The size of the CredentialSpec JSON blob should be limited to avoid abuse. We will limit it to 64K which allows for a lot of room to specify extremely complicated credential specs. Typically this JSON blob is not expected to be more than 1K based on experience so far.
```

Comment:
it amuses me to no end that we literally have our own "64k ought to be enough for anyone" :)